### PR TITLE
Added two DevOps Tools from Kubeshop

### PR DESCRIPTION
**Pull request by evwitmer**
Testkube can help with automated CI/CD pipeline testing and Monokle can help deploy Kubernetes and give a dashboard for working with them.

### Please confirm this pull request meets the following requirements:

- [X ] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [ ] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ X] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [X ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [X ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

>Both of these products are open source devops tools that are growing in popularity.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ X] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Adding Testkube & Monokle to the list of Devops tools because they both help with different areas of the Devops stack. Testkube with Automation of CI/CD pipeline testing and Monokle with Deploying to Kubernetes and having a dashboard for management.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by Sugar25Age** [comment]
Hhss

File: collections/devops-tools/index.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ items:
  - hashicorp/nomad
  - spinnaker/spinnaker
  - Kong/kong
+ - kubeshop/monokle
+ - kubeshop/testkube
```
Comment (MikeMcQuaid) [inline collections/devops-tools/index.md:32]: > expected collection changes to have been appended to the existing item list

Please do this when you get the chance ❤️

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Needs fixes as per CI messaging.

**Review by Eshmove69 — COMMENTED** [review]
Y

**Comment by kenyonj** [comment]
Hi,
Thanks for starting this pull request. In order to review it for acceptance, we need to make sure that all of the prerequisites are satisfied. I see that you didn't check this:

> I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee)

This is a requirement so that we can maintain a high level of independence in this project. Please update if you are able to verify that you meet that requirement.

Thanks!

**Pull request closed without merging**